dame_flame/data_cleaning.py

```diff
@@ -222,6 +218,16 @@ def check_missings(df, df_holdout,  missing_indicator, missing_data_replace,
     '''
     This function deals with all the missing data related stuff
     '''
+    #Ensuring that no entries are non-integers
+    for index, row in df.iterrows():
```

Comment:
Will this line raise an exception if the outcome column is also non-integer? Remember that for our algorithms, the outcome column being continuous is totally ok! It's just the observations that need to be discrete, and we prefer they be integer over discrete floats so that we don't have any floating point overflow issues :)